frontend/utilities/url/index.ts

```diff
@@ -40,26 +54,30 @@ export const buildQueryStringFromParams = (queryParams: QueryParams) => {
   return queryString;
 };
 
-export const reconcileMutuallyExclusiveHostParams = (
-  label?: string,
-  policyId?: number,
-  policyResponse?: string,
-  mdmId?: number,
-  mdmEnrollmentStatus?: string,
-  munkiIssueId?: number,
-  softwareId?: number,
-  osId?: number,
-  osName?: string,
-  osVersion?: string
-): Record<string, unknown> => {
+export const reconcileMutuallyExclusiveHostParams = ({
+  label,
+  policyId,
+  policyResponse,
+  mdmId,
+  mdmEnrollmentStatus,
+  munkiIssueId,
+  lowDiskSpaceHosts,
+  softwareId,
+  osId,
+  osName,
+  osVersion,
+}: IMutuallyExclusiveHostParams): Record<string, unknown> => {
   if (label) {
     return {};
   }
   switch (true) {
     case !!policyId:
       return { policy_id: policyId, policy_response: policyResponse };
     case !!mdmId:
-      return { mdm_id: mdmId, mdm_status: mdmEnrollmentStatus };
```

Comment:
unrelated bug found

Reply:
Pulled solution into a separate PR since this is an unrelated released bug (issue #8215 , PR #8218)